### DISCUSSION TITLE
[FEATURE REQUEST] Saving UI defaults button

### DISCUSSION THREAD

**Discussion by altryne** [hlky/stable-diffusion-webui#368] [Feature suggestions]
For now, just edit [ui defaults](https://github.com/hlky/stable-diffusion-webui/blob/master/webui.yaml)
I agree saving the defaults would be a good feature though

_Originally posted by @hlky in https://github.com/hlky/stable-diffusion-webui/issues/266#issuecomment-1230650076_

**Comment by eremtsov** [comment]
Would be nice to have.
I also found a tiny issue with the config file. 
Image: config file vs the result.
![image](https://user-images.githubusercontent.com/34932866/187313240-d8abeb73-6e6a-4653-b745-3194c0076c4a.png)
The comments on toggles in both the txt2img and the img2img sections seem to be missing "jpg samples" so the numbers after "write sample info" are off by 1.

**Comment by hlky** [comment]
I mean those are just comments but yeah they need to be added

**Comment by eremtsov** [comment]
Yeah, but without them there's no way for a user to know what numbers to use.

**Comment by stevestavropoulos** [comment]
Better option would be to auto-save in a cookie the current user selections, so when you refresh the page you don't have to remake the selections. Every time the user makes a selection (e.g. a tick in a checkbox) the cookie should get updated with the new state.
